File: 13.gilded-rose/src/c#/GildedRose.Tests/GildedRoseTest.cs
using Xunit;

namespace GildedRoseKata.Tests;

public class GildedRoseTest
{
    [Fact]
    public void QualityDecreasesByOneForANormalItemBeforeSellDate()
    {
        var items = new List<Item> { new Item("foo", 10, 20) };
        var app = new GildedRose(items);

        app.UpdateQuality();

        Assert.Equal(9, items[0].SellIn);
        Assert.Equal(19, items[0].Quality);
    }
}

File: 13.gilded-rose/src/php/tests/GildedRoseTest.php
<?php

use Kata\GildedRose;
use Kata\Item;

it('decreases quality by 1 for a normal item before the sell date', function () {
    $gildedRose = new GildedRose([new Item('foo', 10, 20)]);
    $gildedRose->updateQuality();

    $items = $gildedRose->getItems();
    expect($items[0]->sellIn)->toBe(9);
    expect($items[0]->quality)->toBe(19);
});
